src/python/WMCore/JobSplitting/EventAwareLumiBased.py

```diff
@@ -40,6 +40,7 @@ def algorithm(self, *args, **kwargs):
         """
 
         avgEventsPerJob = int(kwargs.get('events_per_job', 5000))
+        jobLimit        = int(kwargs.get('job_limit', -1))
```

Comment:
Using -1 for this is kind of a C thing. Normally you'd use None although you want an int, so how about 0? Then the next bit just becomes ```if jobLimit and ...```